### ISSUE TITLE
Compare GME geom.polygonfetch with lakeFetch

### ISSUE THREAD

**Issue by jsta** [USEPA/lakemorpho#30]
Might be interesting to see if the `geom.polygonfetch` command in Geospatial Modelling Environment returns similar results as `lakeFetch`: http://www.spatialecology.com/gme/geompolygonfetch.htm

**Comment by jhollist** [comment]
Good idea, although I think how they describe that is more akin to Maximum Lake Length than fetch, which uses a bearing to determine the the length.  Their algorithm sounds very similar to what it is implemented in maxLakeLength().  Difference is they rely on the vertices along the polygon edge, whereas maxLakeLength() has the option to specify the number of points along the polygon edge.  But good, I agree that it would be a good check.
